### PR TITLE
LPS-179797

### PR DESCRIPTION
**Pull request by yuhai**
@tinatian, @dantewang, the previous pr is https://github.com/liferay-core-infra/liferay-portal/pull/4089

cc @Kvale1733

### CODE REVIEW

**Comment by Kvale1733** [comment]
ci:test:friendly-url

**Comment by Kvale1733** [comment]
ci:test:upgrade

**Comment by Kvale1733** [comment]
@yuhai No regressions.

File: modules/apps/commerce/commerce-currency-test/src/testIntegration/java/com/liferay/commerce/currency/internal/model/listener/test/CompanyModelListenerTest.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+			int count =
+				_commerceCurrencyLocalService.getCommerceCurrenciesCount(
+					_company.getCompanyId());
+
+			Assert.assertTrue(count > 0);
+
+			String[] commerceCurrencyCodes = {
+				"USD", "AUD", "GBP", "CAD", "CNY", "EUR", "HKD", "JPY", "INR",
+				"BRL"
+			};
```
Comment (tinatian) [inline modules/apps/commerce/commerce-currency-test/src/testIntegration/java/com/liferay/commerce/currency/internal/model/listener/test/CompanyModelListenerTest.java:64]: @yuhai Please try not to hard code, I did not know where these comes from, are commerceCurrencyCodes going to be changed by configuration?

**Comment by tinatian** [comment]
@yuhai The fix had been merged, please rebase and only include the test. Please note, you need to change your commit message as your test is not exposing anything now.

I do not know why default guest public page not created before, the test logic actually make me confused. Please see if there is a more meaningful way to add the test.

If that's too much, leave it to QA to add functional test.

Thanks.
Tina

**Pull request closed without merging**